packages/pyxel/meta.yaml

```
@@ -37,4 +27,4 @@ about:
   home: https://github.com/kitao/pyxel
   PyPI: https://pypi.org/project/pyxel/
   summary: A retro game engine for Python
-  license: MIT
+  license: MIT
```

```suggestion
  license: MIT

```

nit: no newline at the end of file